project-koku/koku
# Sources: Concurrent cleanup

Cleanup the `with ThreadPoolExecutor` statements by consolidating them in `listen_for_messages`.

Testing:
1. add `import time; time.sleep(15)` to `process_message` and `cost_mgmt_msg_filter`
2. send message from sources-api backend
3. After 30 seconds, see message successfully commit

Ensure the `try/except` functionality still works in `listen_for_message`:
1. take DB down.
2. Send sources-api backend message
3. See retry loop.
4. Bring DB up. Wait.
5. See retry loop successfully process message.
